app/services/creators/employments_creator.rb

```diff
@@ -9,12 +9,20 @@ def initialize(assessment_id:, employments_attributes: [])
     end
 
     def call
-      create_records
+      if json_validator.valid?
+        create_records
+      else
+        self.errors = json_validator.errors
```

Comment:
As discussed with KA and CB
```suggestion
        errors.concat(json_validator.errors)
```